Composer/packages/client/src/components/NavLinks/index.tsx

```diff
@@ -21,15 +21,16 @@ export const NavLinks: React.FC<NavLinksProps> = props => {
     <Fragment>
       {navLinks.map(dialog => {
         return (
-          <div
-            css={dialogItem(fileId === dialog.id)}
+          <DefaultButton
             key={dialog.id}
             onClick={() => {
               onSelect(dialog.id);
             }}
-          >
-            {dialog.name}
-          </div>
+            styles={fileId === dialog.id ? dialogItemSelected : dialogItemNotSelected}
+            text={dialog.name}
+            ariaLabel={dialog.id + 'language understanding file'}
```

Comment:
1) this should be translated
2) there probably should be a space between the dialog id and the rest of the text